serenata_toolbox/chamber_of_deputies/official_missions_dataset.py

```diff
@@ -0,0 +1,131 @@
+import os
+import re
+import urllib
+
+from datetime import date, timedelta
+
+import pandas as pd
+import numpy as np
+
+from bs4 import BeautifulSoup
+
+from serenata_toolbox.datasets.helpers import (
+    save_to_csv,
+    translate_column,
+)
+
+
+class OfficialMissionsDataset:
+
+    URL = (
+        'http://www.camara.leg.br/missao-oficial/missao-pesquisa?'
+        'deputado=1&'
+        'nome-deputado=&'
+        'nome-servidor=&'
+        'dati={0}&'
+        'datf={1}&'
+        'nome-evento='
+    )
+
+    def fetch(self, start_date, end_date):
+        """
+        Fetches official missions within the given date range
+        """
+
+        records = []
+        for r in self.__generate_ranges(start_date, end_date):
+            if os.environ.get('DEBUG') == '1':
+                print(r)
+            for record in self.__fetch_missions_for_range(r[0], r[1]):
+                records.append(record)
```

Comment:
In a simplistic way: `yield from range(42)` is just sugar syntax for:

```python
for number in range(42):
   yield number
```
As you've said `records` list is pretty small no need for that… can you send the dataset so I add it to S3 and merge this?